src/ui/Menu/styles.ts

```diff
@@ -3,47 +3,40 @@ import tw, { css } from 'twin.macro';
 /**
  * Styles applied to MenuItem and MenuLink
  */
-const menuItemStyle = css`
+export const menuItemStyle = css`
```

Comment:
can we group everything in an object and export it only?
```ts
export const menuStyles = {
  item: css``
}
```

Reply:
Aren't all of them exported anyway? Just a way to give them a namespace.

Reply:
We can put the styles in its own component where they're used and get rid of the `styles.ts`. I think I found a way to share it between the MenuItem and the MenuLink.

Reply:
Refactored the styles into its own component and introduced the `MenuItemBase`